crates/bevy_render/src/maths.wgsl

```diff
@@ -63,14 +63,16 @@ fn mat4x4_to_mat3x3(m: mat4x4<f32>) -> mat3x3<f32> {
     return mat3x3<f32>(m[0].xyz, m[1].xyz, m[2].xyz);
 }
 
-// Creates an orthonormal basis given a Z vector and an up vector (which becomes
-// Y after orthonormalization).
+// Creates an orthonormal basis given a normalized Z vector.
 //
 // The results are equivalent to the Gram-Schmidt process [1].
 //
 // [1]: https://math.stackexchange.com/a/1849294
-fn orthonormalize(z_unnormalized: vec3<f32>, up: vec3<f32>) -> mat3x3<f32> {
-    let z_basis = normalize(z_unnormalized);
+fn orthonormalize(z_basis: vec3<f32>) -> mat3x3<f32> {
```

Comment:
Maybe calling this z_normalized would help indicate that it should be normalised.